packages/console/src/hooks.tsx

```diff
@@ -12,15 +12,19 @@ type UsePlanResult = SWRResponse<PlanGetSuccess | undefined> & {
 }
 
 export const usePlan = (account: Account) => {
-  const result = useSWR<PlanGetSuccess | undefined>(planKey(account), {
+  const result = useSWR(planKey(account), {
     fetcher: async () => {
-      if (!account) return
+      console.log(`Loading plan for account ${account.toEmail()}...`)
       const result = await account.plan.get()
-      if (result.error) throw new Error('getting plan', { cause: result.error })
+      if (result && result.error) {
+        console.log(`Failed to get plan: ${result.error.toString()}`)
+        return result.error
```

Comment:
I know we worked on this together this morning but I'm realizing this is probably wrong - if you return the error here it will get passed back up as the `data` value of the hook return value, which is not what we want - I think it should actually do exactly what it was doing before, and we should actually be checking `error?.cause?.name` (or something like that) above

Reply:
Yup. I just reverted and synced with main. All good now.